2.4/root/usr/bin/run-mongod-pet

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+#
+# Run mongod as part of PetSet.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+source ${CONTAINER_SCRIPTS_PATH}/common.sh
+
+function usage() {
+  echo "You must specify the following environment variables:"
+  echo "  MONGODB_USER"
+  echo "  MONGODB_PASSWORD"
+  echo "  MONGODB_DATABASE"
+  echo "  MONGODB_ADMIN_PASSWORD"
+  echo "  MONGODB_KEYFILE_VALUE"
+  echo "  MONGODB_REPLICA_NAME"
+  echo "Optional variables:"
+  echo "  MONGODB_SERVICE_NAME (default: mongodb)"
+  echo "MongoDB settings:"
+  echo "  MONGODB_NOPREALLOC (default: true)"
+  echo "  MONGODB_SMALLFILES (default: true)"
+  echo "  MONGODB_QUIET (default: true)"
+  exit 1
+}
+
+# Make sure env variables don't propagate to mongod process.
+function unset_env_vars() {
+  unset MONGODB_USER MONGODB_PASSWORD MONGODB_DATABASE MONGODB_ADMIN_PASSWORD
+}
+
+function cleanup() {
+  echo "=> Shutting down MongoDB server ..."
```

Comment:
> If an admin wants to change the topology permanently, either scale up or down, he needs to manually reconfigure the replica set (we might facilitate that in the future, but not a common use case).

So the cool buttons (up/down arrows in project overview) won't work?

> The older scripts try to deal with IP changes, removing old IPs and adding new IPs to the config dynamically, and we know how many corner cases that has :)
(e.g., https://bugzilla.redhat.com/show_bug.cgi?id=1260586)

You wrote in that bug "However, the centos image works as expected.". What is a difference to RHEL image?
Does not #186 fix it?

I am not claiming that current scripts have no bug and there is a lot of space for improvements.